src/Support/Utils/DirectiveUtil.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Utils;
```

Comment:
This could be in src/Schema/Utils? I am not a fan of the Support directory, since it tells us pretty much nothing about what the stuff in there actually does